designsystem/src/main/kotlin/com/natura/android/checkbox/CheckBox.kt

```diff
@@ -9,4 +9,50 @@ class CheckBox @JvmOverloads constructor(
     context: Context,
     private val attrs: AttributeSet? = null,
     defStyleAttr: Int = R.attr.checkboxPrimary
-) : AppCompatCheckBox(context, attrs, defStyleAttr)
+) : AppCompatCheckBox(context, attrs, defStyleAttr) {
```

Comment:
Talvez caiba uma análise do JvmOverloads, pois em alguns casos, ele tem um comportamento diferente do esperado, especialmente em custom views - (https://zsmb.co/jvmoverloads-for-android-views/)